test/programs/DiagrammaticPrograms.jl

```diff
@@ -334,8 +315,10 @@ M′ = @migration SchGraph SchSet begin
   V => @unit
   E => X
 end
-# something weird about == for datamigrations
-@test functor(M′) == functor(M) && M′.params == M.params
+@test M′ isa DataMigrations.ConjSchemaMigration
+@test isempty(shape(ob_map(functor(M′), :V)))
+# XX:This one has vector maps in the empty shape_map which isn't right
```

Comment:
Is this comment up to date? Is it pointing to a bug?

Reply:
Out of date!